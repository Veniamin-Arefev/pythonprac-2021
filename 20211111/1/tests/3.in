A, B, C = OrderedComplex(123.2, -321), OrderedComplexMul(44, 123),  OrderedComplexMul(5643.8, 123)
print(A < B, B @ B, OrderedComplex(C @ C, B @ B) < A)

