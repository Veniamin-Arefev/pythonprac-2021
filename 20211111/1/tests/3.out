False 17065 False
